packages/core/src/resources/Namespaces.ts

```diff
@@ -37,12 +42,12 @@ export class Namespaces<C extends boolean = false> extends BaseResource<C> {
   }
 
   exists<E extends boolean = false>(
-    namespaceId: string | number,
+    namespace: string | number,
```

Comment:
If you can remove the number type here, I can merge this ASAP 🙏

Reply:
done! :D